docs/pages/reference/cli.mdx

```diff
@@ -2260,6 +2260,60 @@ $ tctl get clusters,users
 
 </ScopedBlock>
 
+### tctl edit
+
+Modify a Teleport resource using your preferred text editor.
+
+```code
+$ tctl edit [<resource-type/resource-name>]
+```
+
+The text editor is selected by checking the following, in order of precedence:
+
+- the `TELEPORT_EDITOR` environment variable
+- the `VISUAL` environment variable
+- the `EDITOR` environment variable
+- defaulting to `vi`
+
+`tctl` will fetch the resource from the backend and open it in the selected editor.
+When the editor process terminates, `tctl` will push the updates to the Teleport cluster.
+To abandon the edit, close the editor without saving the file.
+
+Some graphical editors like VS Code launch a background process rather than running
+in the foreground. This prevents `tctl` from properly detecting when the editor
+process terminates. To work around this, check whether your editor supports a "wait"
+option that waits for files to be closed before terminating. For example, to edit a
+Teleport resource with VS Code, you can set `TELEPORT_EDITOR="code --wait"`.
+
+**Note:** Renaming resources with `tctl edit` is not supported since Teleport resources
+often refer to other resources by name. To rename a resource, we recommend:
+- fetch the resource with `tctl get` and redirect the output to a file
+- change the name of the resource in the file
+- save the new resource with `tctl create -f`
+- update any references to the old resource
+
+#### Arguments
+
+- `[<resource-type/resource-name>]` Resource to get
+  - `<resource type>` Type of a resource \[for example: `user,cluster,token`]
+  - `<resource name>` Resource name to edit
```

Comment:
```suggestion
- `[<resource-type/resource-name>]` Theresource to get.
  - `<resource type>` The type of a resource \[for example: `user,cluster,token`].
  - `<resource name>` The resource name to edit.
```

Reply:
Accepted the `The` suggestion, but omitted the closing period because it's inconsistent with the rest of this page

Reply:
OK, at some point I'll make an effort to fix up the formatting of the entire page... one day.